firmware_configurations/klipper/Mellow/FLY-Super8/readme.md

```diff
@@ -0,0 +1,13 @@
+Klipper configuration of FLY-Super8 motherboard for Voron2.4
+=========
+
+## EN
+![Pins](pins.jpg)
+
+[EN Docs](https://teamgloomy.github.io/fly_super8_pins.html)
+
+
+## zh_CN
+![cn v2.4 pins](voron2.4-ZH_cn.jpg)
+
+[CN Docs](http://fly3d.org/docs/#/board/fly_super8/)
```

Comment:
This link returns a 404 instead of the documents.

Reply:
Thank you very much for pointing out my mistakes. I have changed and submitted new content.
thanks